docs(docs-infra): update /deep/ to ::ng-deep

andyg3 opened the pull request:
Since, for the time being, ::ng-deep is the preferred syntax, the example should be updated.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
Example uses :host /deep/

Issue Number: N/A


## What is the new behavior?
Example uses :host ::ng-deep

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

andyg3 committed:
Update /deep/ to ::ng-deep

Since, for the time being, ::ng-deep is the preferred syntax, the example should be updated.

petebacondarwin reviewed (approved):
Thanks @andyg3.

Please update the commit message to comply with our requirements. I see you updated the PR title but this is not enough.

andyg3 committed:
docs(docs-infra): update /deep/ to ::ng-deep

Since, for the time being, ::ng-deep is the preferred syntax, the example should be updated.
